lib/util/utils.ts

```diff
@@ -246,9 +247,9 @@ export function executePromisesSequentially(promiseFactories: Array<any>, kickst
  * @return {boolean} boolean.
  */
 export function isIotHubConnectionString(connectionString: string): boolean {
-  if (!connectionString || typeof connectionString !== "string") {
-    throw new Error("connectionString is a required parameter and must be of type string.");
-  }
+  throwTypeErrorIfParameterMissing("connectionString", connectionString);
+  connectionString = String(connectionString);
```

Comment:
We can skip the parameter missing check here.
If none was provided, connectionString will be `undefined` and `parseConnectionString` will give appropriate errors